manifests/package.pp

```diff
@@ -60,10 +60,11 @@
   }
 
   package { $pkg_title:
-    ensure  => $sensu::version,
-    name    => $pkg_name,
-    source  => $pkg_source,
-    require => $pkg_require,
+    ensure          => $sensu::version,
+    name            => $pkg_name,
+    source          => $pkg_source,
+    install_options => ['--allow-unauthenticated'],
```

Comment:
You might want to reconsider this and sign your repository instead of making the default less secure for everyone.